index.html

```diff
@@ -31,6 +33,143 @@ <h2 class="connect">LET'S CONNECT</h2>
+            <img src="./images/Snapshoot Portfolio3.png" class="snapshot"/>
+    
+            <h3 class="tonic">Tonic</h3>
+    
+            <ul class="tags">
+                <li class="tags-li-blod">Canopy</li>
+                <li class="tags-li">Back End Dev</li>
+                <li class="tags-li">2023</li>
+            </ul>
+    
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stack">
+                <li class="stack-elem">HTML</li>
+                <li class="stack-elem">CSS</li>
+                <li class="stack-elem">Javascript</li>
+            </ul>
+           <button class="see-project">See project</button>
+            </div>
+    
+        </article>
+        
+
+        <article class="works">
+            <img src="./images/Snapshoot Portfolio4.png" class="snapshot"/>
+    
+            <h3 class="tonic">Tonic</h3>
+    
+            <ul class="tags">
+                <li class="tags-li-blod">Canopy</li>
+                <li class="tags-li">Back End Dev</li>
+                <li class="tags-li">2023</li>
+            </ul>
+    
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stack">
+                <li class="stack-elem">HTML</li>
+                <li class="stack-elem">CSS</li>
+                <li class="stack-elem">Javascript</li>
+            </ul>
+           <button class="see-project">See project</button>
+            </div>
+    
+        </article>
+        
+    </section>
+
+
+    <section class="About-myself">
+        <h2 id="myself">About Myself</h2>
+        <p class="pg2">Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t
+            hesitate to contact me.</p>
+            <div class="middle-sec">
+        <h2 class="connect">LET'S CONNECT</h2>
+        <ul class="footer">
+            <li><a href="#"><img src="images/twitter.png" /></a></li>
+            <li><a href="#"><img src="images/github.png" /></a></li>
+            <li><a href="#"><img src="images/Linkedin icon.png" /></a></li>
+            <li><a href="#"><img src="images/medium.png" /></a></li>
+            <li><a href="#"><img src="images/angellist.png" /></a></li>
+        </ul>
+        <button class="resume">get my resume</button>
+      </div class="middle-sec">
+        <ul class="skills">
+            <li class="dropdown">
+              <a href="#" class="dropdown-btn"><div>Languages</div><img src="images/arrow.png" class="rotate"></a>
+
+              <div class="dropdown-content">
+                <a href="#"><img src="images/Ellipse for JS.png">Javascript</a>
+                <a href="#"><img src="images/htm.png">HTML</a>
+                <a href="#"><img src="images/cs.png">CSS</a>
+              </div>
```

Comment:
- [ Optional] Well done adding the languages as a dropdown menu however, you have to stick to the Figma design as much as possible.